### ISSUE TITLE
Mapping -- storing mapping results in cache

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#439]
Every time the mapping button is clicked, the results of the mapping are stored in the database under seed_buildingsnapshot. If the mappings are not saved, the results are still stored. I suggest we store the mapping results in the cache to save space/time.

preferable we update the serializer to allow for unpickled data (i.e. JSON data) to be stored

(RM -- I am adding this issue based on our Oct 9 meeting; see Issue #291 which is for backgound and which I am going to close)

**Comment by RDmitchell** [comment]
@nllong -- this is still outstanding, right? Do you think you will have time to change this for the Feb release? If not, I will move the Milestone date.

**Comment by RDmitchell** [comment]
This can really make a difference if you are trying to map a file with a lot of records. I have a sample data file with 12,000+ records, and if I try to map it a second time, there is a real delay in getting the records to appear in the final mapping screen. And when I go to delete the buildings (that don't actually appear in the building record at this point) as a superuser, the program takes a while to delete them, so they are actually being stored, even though it is not obvious from the UI.

**Comment by nicholasserra** [comment]
I don't know whether this is going to work as you're thinking. Those buildings in the DB are then directly used once mapping is done, I don't think we're going to be able to change the storage of them without totally rewriting a lot of the building logic. Sounds like a fix for #431 might make this a little smoother.

**Comment by RDmitchell** [comment]
Yes, let's work on #431 first, then come back to this and see if it still makes sense.

**Comment by RDmitchell** [comment]
am closing as this is presumably no longer relevant
